locust/main.py

```diff
@@ -392,6 +399,11 @@ def main():
         logger.error("Locust can not run distributed with the web interface disabled (do not use --no-web and --master together)")
         sys.exit(0)
 
+    if options.no_reset_stats:
+        runners.RESET_STATS_AFTER_HATCHING = False
+    if not runners.RESET_STATS_AFTER_HATCHING:
```

Comment:
I removed the log message all together, and changed so that we don't use the module level variable for controlling the reset behaviour. Let me know if you think that's a bad idea.